packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue

```diff
@@ -121,80 +121,52 @@
 			</tera-slider-panel>
 			<tera-slider-panel v-model:is-open="isOutputOpen" header="Output" content-width="100%">
 				<template #content>
-					<header class="flex align-items-center p-3">
-						<h4>Equation conversions</h4>
-						<Button
-							v-if="selectedModel"
-							label="Save for re-use"
-							outlined
-							severity="secondary"
-							class="ml-auto"
-							@click="showSaveModelModal = true"
-						/>
-					</header>
 					<tera-drilldown-preview>
-						<tera-model-description v-if="selectedModel" :model="selectedModel" :generating-card="isGeneratingCard" />
-						<tera-operator-placeholder v-else :node="node" style="height: 100%" />
+						<tera-model v-if="selectedModel" :assetId="selectedModel.id" @on-save="onModelSaveEvent" />
+						<tera-operator-placeholder v-else :node="node" class="h-100" />
 					</tera-drilldown-preview>
 				</template>
 			</tera-slider-panel>
 		</template>
 	</tera-drilldown>
-	<tera-save-asset-modal
-		v-if="selectedModel"
-		:asset="selectedModel"
-		:is-visible="showSaveModelModal"
-		@close-modal="onCloseModelModal"
-	/>
 </template>
 
 <script setup lang="ts">
 import { AssetBlock, WorkflowNode } from '@/types/workflow';
 import Checkbox from 'primevue/checkbox';
+import Textarea from 'primevue/textarea';
+import TeraAssetBlock from '@/components/widgets/tera-asset-block.vue';
 import TeraDrilldown from '@/components/drilldown/tera-drilldown.vue';
 import TeraDrilldownPreview from '@/components/drilldown/tera-drilldown-preview.vue';
-import TeraAssetBlock from '@/components/widgets/tera-asset-block.vue';
 import { computed, onMounted, ref, watch } from 'vue';
-import { downloadDocumentAsset, getDocumentAsset, getDocumentFileAsText } from '@/services/document-assets';
 import type { Card, DocumentAsset, Model } from '@/types/Types';
 import { cloneDeep, isEmpty } from 'lodash';
 import { equationsToAMR, type EquationsToAMRRequest } from '@/services/knowledge';
-import Button from 'primevue/button';
```

Comment:
There are `Button` components here.  Im getting something that looking like this for `model from equations`
<img width="1512" alt="Screenshot 2024-10-02 at 2 14 27 PM" src="https://github.com/user-attachments/assets/2bf98d36-5cf7-4b1f-bf1a-fad3cd0e7643">

Reply:
Great catch